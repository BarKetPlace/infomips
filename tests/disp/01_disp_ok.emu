#-------
#TEST_RETURN_CODE=PASS           
#TEST_COMMENT="Teste fonctionnement standard"
#-------


load ./elf/boucle.o

disp mem map

disp mem 0x12:0x10

disp mem 0x0:0xc
disp mem 0x0 0x4 0x8 0xc 0xf
disp mem 0x0+28
disp mem 0x0:0x1c


disp reg all
disp reg $t0 $v0

exit
