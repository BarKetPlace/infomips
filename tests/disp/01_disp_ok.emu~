#-------
#TEST_RETURN_CODE=PASS           
#TEST_COMMENT="Teste fonctionnement standard"
#-------


load ./elf/boucle.o

disp mem map

disp mem 0x12:0x10
disp mem 0x0+24
disp mem 0x0:0xc
disp mem 0x0 0x4 0x8 0xc 0xf


disp reg all

disp reg $t0 $v0

