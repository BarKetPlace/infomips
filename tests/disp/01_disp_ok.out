0x00000026	0xfc0007ff	XOR	R	3	rd rs rt 
0x0000000c	0xfc00003f	SYSCALJ	J	0	
0xac000000	0xfc000000	SW	I	2	rt imm 
0x00000023	0xfc0007ff	SUBU	R	3	rd rs rt 
0x00000022	0xfc0007ff	SUR	R	3	rd rs rt 
0x00000002	0xffe0003f	SRL	I	3	rd rt imm 
0x00000003	0xffe0003f	SRA	I	3	rd rt imm 
0x0000002b	0xfc0007ff	SLTU	R	3	rd rs rt 
0x2c000000	0xfc000000	SLTIU	I	3	rt rs imm 
0x28000000	0xfc000000	SLTI	I	3	rd rs imm 
0x0000002a	0xfc0007ff	SLT	R	3	rd rs rt 
0x00000000	0xffe0003f	SLL	I	3	rd rt imm 
0x7c000420	0xffe007ff	SEB	R	2	rd rt 
0xa0000000	0xfc000000	SB	I	2	rt imm 
0x34000000	0xfc000000	ORI	I	3	rt rs imm 
0x00000025	0xfc0007ff	OR	R	3	rd rs rt 
0x00000012	0xffff07ff	MFLO	R	1	rd 
0x00000018	0xfc00ffff	MULT	R	2	rs rt 
0x00000010	0xffff07ff	MFHI	R	1	rd 
0x8c000000	0xfc000000	LW	I	2	rt imm 
0x3c000000	0xffe00000	LUI	I	2	rt imm 
0x90000000	0xfc000000	LBU	I	2	rt imm 
0x80000000	0xfc000000	LB	I	2	rt imm 
0x00000008	0xfc1ff83f	JR	R	1	rs 
0x00000009	0xfc1f003f	JALR	R	2	rd rs 
0x0c000000	0xfc000000	JAL	J	1	target 
0x0000001a	0xfc00ffff	DIV	R	2	rs rt 
0x0000000d	0xfc00003f	BREAK	J	0	
0x14000000	0xfc000000	BNE	I	3	rs rt imm 
0x04000000	0xfc1f0000	BLTZ	I	2	rs imm 
0x18000000	0xfc1f0000	BLEZ	I	2	rs imm 
0x1c000000	0xfc1f0000	BGTZ	I	2	rs imm 
0x04010000	0xfc1f0000	BGEZ	I	2	rs imm	 
0x10000000	0xfc000000	BEQ	I	3	rs rt imm 
0x03000000	0xfc000000	ANDI	I	3	rs rt imm 
0x00000024	0xfc0007ff	AND	R	3	rd rs rt 
0x00000021	0xfc0007ff	ADDU	R	3	rd rs rt 
0x24000000	0xfc000000	ADDIU	I	3	rt rs imm 
0x11111111	0xffffffff	NOP	J	0	
0x08000000	0xfc000000	J	J	1	target 
0x02000000	0xfc000000	ADDI	I	3	rt rs imm 
0x00000020	0xfc0007ff	ADD	R	3	rd rs rt 
Virtual memory map (4 segments)
.text   	r-x	Vaddr: 0x00000000	Size: 24 bytes
.data   	rw-	Vaddr: 0x00001000	Size: 4 bytes
.bss    	rw-	Vaddr: 0x00002000	Size: 0 bytes
[STACK] 	rw-	Vaddr: 0xfffff000	Size: 8388608 bytes
0x00000010 00 00
0x00000000 20 09 00 0c 00 00 50 20 21 4a 00 01
0x00000004 :  00
0x00000008 :  21
0x0000000c :  15
0x0000000f :  00
zero	: 0x00000000	at	: 0x00000000	v0	: 0x00000000	v1	: 0x00000000
v1	: 0x00000000	a0	: 0x00000000	a1	: 0x00000000	a2	: 0x00000000
a3	: 0x00000000	t0	: 0x00000000	t1	: 0x00000000	t2	: 0x00000000
t3	: 0x00000000	t4	: 0x00000000	t5	: 0x00000000	t6	: 0x00000000
t7	: 0x00000000	s0	: 0x00000000	s1	: 0x00000000	s2	: 0x00000000
s3	: 0x00000000	s4	: 0x00000000	s5	: 0x00000000	s6	: 0x00000000
s7	: 0x00000000	t8	: 0x00000000	t9	: 0x00000000	k0	: 0x00000000
k1	: 0x00000000	gp	: 0x00000000	sp	: 0xfffff000	fp	: 0x00000000
ra	: 0x00000000	pc	: 0x00000000	HI	: 0x00000000	
t0	0x00000000
v0	0x00000000
